### DISCUSSION TITLE
Snowflake support?

### DISCUSSION THREAD

**Discussion by omar33** [prisma/prisma#11644] [Q&A] [answered]
I see a growing list of supported databases which is great. I am curious if support for snowflake is coming anytime soon?

**Comment by janpio** [comment, marked as answer]
Not planned right now. Actually we do not even have a feature request for Snowflake support, so you might want to create that. Make sure to include a description of your use case.

(I am working with a startup right now that might offer connecting to Snowflake through a PostgreSQL proxy, so maybe that might even be enough.)

**Reply by chris-activewrite** [reply]
Can you share info about that startup?

**Reply by janpio** [reply]
They unfortunately pivoted away from that idea.

But we do have a feature request now: https://github.com/prisma/prisma/issues/11881

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
